vms/platformvm/txs/executor/subnet_tx_verification.go

```diff
@@ -43,6 +43,14 @@ func verifyPoASubnetAuthorization(
 		return nil, err
 	}
 
+	_, _, err = chainState.GetSubnetManager(subnetID)
+	if err == nil {
+		return nil, fmt.Errorf("%q %w", subnetID, errIsImmutable)
+	}
+	if err != database.ErrNotFound {
+		return nil, err
+	}
+
```

Comment:
I think this is probably the most obvious way to do this change versus how I had it originally (cc @marun)